test/evergreen.yml

```diff
@@ -5875,6 +5885,10 @@ buildvariants:
     - name: examples-c-production-disable-static-test
     - name: format-stress-pull-request-test
     - name: make-check-test
+      distros: ubuntu2004-large
+    - name: csuite-long-running
+      distros: ubuntu2004-large
+      exec_timeout_secs: 21600 # 6 hrs
```

Comment:
Should we consider lower the running frequency for the long csuite tests (using `batchtime` setting)?

Reply:
Yes, I wan wondering that but not sure how we determine what frequency to use?

Reply:
I think running it "once per day" is good enough.